src/hooks/useHandleRedeem.tsx

```diff
@@ -585,6 +610,32 @@ export function useHandleRedeem() {
   const seiAddress = seiWallet?.getAddress();
   const signedVAA = useTransferSignedVAA();
   const isRedeeming = useSelector(selectTransferIsRedeeming);
+  const sourceChain = useSelector(selectTransferSourceChain);
+  const sourceAsset = useSelector(selectTransferSourceAsset);
+  const sourceParsedTokenAccount = useSelector(
+    selectTransferSourceParsedTokenAccount
+  );
+  const targetAddressHex = useSelector(selectTransferTargetAddressHex);
+
+  const onSuccess = useCallback(() => {
+    telemetry.on.transferSuccess({
+      fromChainId: sourceChain,
+      toChainId: targetChain,
+      fromTokenSymbol: sourceAsset,
+      toTokenSymbol: undefined,
+      fromTokenAddress: undefined,
+      toTokenAddress: !!sourceParsedTokenAccount?.isNativeAsset
+        ? "native"
+        : targetAddressHex,
```

Comment:
Similar change as the useHandleTransfer:
```suggestion
      fromTokenSymbol: sourceParsedTokenAccount?.symbol,
      toTokenSymbol: targetParsedTokenAccount?.symbol,
      fromTokenAddress: !!sourceParsedTokenAccount?.isNativeAsset
        ? "native"
        : sourceAsset,
      toTokenAddress: !!targetParsedTokenAccount?.isNativeAsset
        ? "native"
        : targetAddressHex,
```